### ISSUE TITLE
BRAND-NEW bug in v14.0.0

### ISSUE THREAD

**Issue by TheTT** [linsyking/Messenger#17]
**Environment** 

- OS: Windows 11
- Web browser: Firefox (127.0.2), Chrome (126.0.6478.127) and Edge (126.0.2592.81)

**Detailed explanation of the problem**

Under Firefox, some values in messenger are not properly initialized, such as the real width and height of the web-page. These values are 0 when the page is opened, however they returned to normal after changing the window size. This problem prevents all functions involving `posToReal` and `lengthToReal` from working properly. This was not the case in previous versions of messenger.  
I think this is due to a change in `Messenger.UI.Init.init`. Comparing `Line 91~94 in 14.0.0/src/Messenger/UI/Init.elm` and `Line 93~94 in 12.1.0/src/Messenger/UI/Init.elm`, we can conclude that 14.0.0 should rely on the "change window size" message sent by the browser when initializing a web page. Although Chrome and Edge do this, it seems that Firefox does not. I think this incompatibility with some browsers is dangerous and should be fixed ASAP.

**Comment by yYuanQ** [comment]
I also meet this problem with the same version of Firefox.

**Comment by linsyking** [comment]
Yes, in 14 we move the logic of initializing the viewport size to Elm rather than JS.
We used [getViewport](https://package.elm-lang.org/packages/elm/browser/latest/Browser-Dom#getViewport) from Elm but it seems they wonn't work properly on Firefox.

**Comment by linsyking** [comment]
We tried to fix this problem by using `scene` size. You may try using `14.0.2`.
